### DISCUSSION TITLE
Excessive De-pressurisation

### DISCUSSION THREAD

**Discussion by ShehabGamrah** [firemodels/fds#13967] [Q&A]
[PRESS.txt](https://github.com/user-attachments/files/18283606/PRESS.txt)

I am getting -2500 pressure values. What is causing this error.

**Comment by drjfloyd** [comment]
This is a complex file that is going to take time for anyone but you to understand especially when you provide no details about what you are trying to do and what you have ruled out yourself as possible causes.

What version of FDS are you using?
Where are you seeing -2500 Pa of pressure both location and which specific output?
Are you seeing this at the start, slowly over the simulation, or suddenly later on in the simulation? Whatever you answer should provide you with a clue as to what might be happening.
You have a lot control functions opening and closing doors.  Have you checked to make sure they are correct and that your HOLE inputs fully cut through the OBST?

**Comment by ShehabGamrah** [comment]
The model consists of a stairwell, lobby, corridor and fire compartment. When the extract operates  i am getting over -2000Pa. Moreover plotting the HRR.csv file I am getting this in one zone out of zones don't know if this could help debugging. But I have run similar models without this problem so don't know why it is happening here.

**Comment by mcgratta** [comment]
I will run the case.

**Reply by ShehabGamrah** [reply]
Many Thanks

**Comment by ShehabGamrah** [comment]
![{6D576484-903A-4FAA-B432-0E0E23FBA100}](https://github.com/user-attachments/assets/15723d00-1783-4229-b74f-d0c476555ca9)
I have re-run the simulation by removing wall at this location and it seems to work. This wall is contained in the previously attached .txt file. This wall contains a door, so how can I re-add it while not causing this excessive pressures. I think this could be related to the way automatic zones are formed as in the .hrr file there was excessive de-pressure in this zone.

**Comment by mcgratta** [comment]
I ran the case with the latest source code. I see this after 500 s
![image](https://github.com/user-attachments/assets/b705df37-652e-4d1f-959f-b212380d0e73)

I do not see where the pressure drops to -2500 Pa.

**Reply by ShehabGamrah** [reply]
I am using FDS 6.8.0. I have removed the wall separating the lobby and the problem seems sensitive to the wall removed (black circle attached previous image). When I checked the .HRR the pressure problem seems in one of the zones. Is there a way to control the zones within FDS 6.8.0 to re-add the lobby wall.

**Reply by mcgratta** [reply]
I am not sure what you mean.

**Reply by drjfloyd** [reply]
@ShehabGamrah your solution is update to the current release and try that.
